src/app/components/shared/demo-page-plunker-service.ts

```diff
@@ -178,6 +189,10 @@ ${imports.join('\n')}
 
 import { AppComponent } from './app.component';
 
+// This is temporary to get list-view-grid functioning properly
+// A future relese of SKY UX components would fix this.
```

Comment:
"relese"

Also, might be inconsequential, but would this comment appear in the plunker's source files? Again, not super important, just curious if we want this language on our plunkers.